content/en/docs/languages/go/generated-code.md

```diff
@@ -3,6 +3,7 @@ title: Generated-code reference
 linkTitle: Generated code
 weight: 95
 ---
+The following documents the latest generated code guide which uses generics.  For documentation about the older version of generated code that did not use generics, please see the doc for [old generated code](/docs/languages/go/generated-code-old).
```

Comment:
Call out that this version of the generated code is the default, and that the old documentation is available for people who are looking at old generated code, and that it's temporarily possible to get this behavior by setting the flag.

Reply:
Done.